### PR TITLE
(docs) : seo changes for user guides

### PR DESCRIPTION
**Pull request by rakeshPRaghu**
1. added keywords and description from the docs

Signed-off-by: Rakesh PR <rakesh.pr@mayadata.io>

### CODE REVIEW

File: docs/versioned_docs/version-2.11.0/user-guides/uninstall.md [hunk 1]
```diff
@@ -1,6 +1,10 @@
 ---
 id: uninstall
 title: Uninstalling OpenEBS
+keywords:
+ - Uninstalling OpenEBS
+ - Uninstall OpenEBS
+ - Deletion of Jiva Volumes
 ---
```
Comment (Pallavi-PH) [inline docs/versioned_docs/version-2.11.0/user-guides/uninstall.md:8]: description needs to be added here

File: docs/versioned_docs/version-2.11.0/user-guides/upgrade.md [hunk 2]
```diff
@@ -1,6 +1,11 @@
 ---
 id: upgrade
 title: Upgrading OpenEBS
+keywords:
+ - Upgrading OpenEBS
+ - OpenEBS upgrade
+ - Supported upgrade paths
+description: Upgrade to the latest OpenEBS 2.11.0 version is supported only from 1.0.0 and later.
```
Comment (Pallavi-PH) [inline docs/versioned_docs/version-2.11.0/user-guides/upgrade.md:8]:
```suggestion
description: Upgrade to the latest OpenEBS 2.11.0 version is supported only from v1.0.0 and later.
```

File: docs/versioned_docs/version-2.11.0/user-guides/localpv-device.md [hunk 3]
```diff
@@ -1,6 +1,18 @@
 ---
 id: localpv-device
 title: OpenEBS Local PV Device User Guide
+keywords:
+ - OpenEBS Local PV Device
+ - Local PV Prerequisites
+ - OpenEBS Local PV Installation
+ - Create StorageClass
+ - Create StorageClass
```
Comment (Pallavi-PH) [inline docs/versioned_docs/version-2.11.0/user-guides/localpv-device.md:9]: This is repeated

File: docs/versioned_docs/version-2.11.0/user-guides/prerequisites.mdx [hunk 4]
```diff
@@ -1,6 +1,13 @@
 ---
 id: prerequisites
 title: OpenEBS Install Prerequisites
+keywords:
+ - OpenEBS Install Prerequisites
+ - prerequisites
```
Comment (Pallavi-PH) [inline docs/versioned_docs/version-2.11.0/user-guides/prerequisites.mdx:6]: These both are the same right?

File: docs/versioned_docs/version-2.11.0/user-guides/uninstall.md [hunk 5]
```diff
@@ -1,6 +1,10 @@
...
 id: uninstall
 title: Uninstalling OpenEBS
+keywords:
+ - Uninstalling OpenEBS
+ - Uninstall OpenEBS
```
Comment (Pallavi-PH) [inline docs/versioned_docs/version-2.11.0/user-guides/uninstall.md:6]: Repeating

**Review by satyapriyamishra222 — APPROVED** [review]

**Review by IsAmrish — APPROVED** [review]

**Review by Pallavi-PH — APPROVED** [review]

**Review by satyapriyamishra222 — APPROVED** [review]

**Pull request merged**